pallets/subtensor/src/migration.rs

```diff
@@ -474,3 +488,48 @@ pub fn migrate_to_v2_fixed_total_stake<T: Config>() -> Weight {
         Weight::zero()
     }
 }
+
+pub fn migrate_to_v7_weight_commits<T: Config>() -> Weight {
```

Comment:
This should be implemented with `OnRuntimeUpgrade` ,and also added to the runtime (try-runtime checks should fail otherwise)

Reply:
done